`kitchen-sink` example fails on tests

### What version of Turborepo are you using?

latest

### What package manager are you using / does the bug impact?

Yarn v1

### What operating system are you using?

Mac

### Describe the Bug

When running `yarn test` on `examples/kitchen-sink` fails with:

```
ui:test:     src/__tests__/index.test.tsx:7:17 - error TS2584: Cannot find name 'document'. Do you need to change your target library? Try changing the 'lib' compiler option to include 'dom'.

ui:test:     7     const div = document.createElement("div");
ui:test:                       ~~~~~~~~

ui:test: Test Suites: 1 failed, 1 total
ui:test: Tests:       0 total
ui:test: Snapshots:   0 total
ui:test: Time:        2.321 s
ui:test: Ran all test suites.
ui:test: error Command failed with exit code 1.
ui:test: Error: command finished with error: exit status 1
child_process.js:830
    throw err;
    ^
```


### Expected Behavior

Tests should pass

### To Reproduce

* Download `examples/kitchen-sink`
* `cd examples/kitchen-sink && yarn install && yarn test`
*

--- a/examples/kitchen-sink/packages/ui/tsconfig.json
+++ b/examples/kitchen-sink/packages/ui/tsconfig.json
@@ -1,4 +1,7 @@
 {
+  "compilerOptions": {
+    "lib": ["dom", "ES2015"],
+  },
   "extends": "tsconfig/react-library.json",
   "include": ["."],
   "exclude": ["dist", "build", "node_modules"]

